packages/tailwind/src/tailwind-presets.ts

```diff
@@ -18,29 +20,29 @@ export const kaizenTailwindTheme: KaizenTailwindTheme = {
     ...defaultTheme.dataViz,
   },
   spacing: {
-    ...defaultTheme.spacing,
-    ...twDefaultTheme.spacing,
+    ...kzSpacing,
   },
```

Comment:
You could actually do this to save a few lines :) Not against it if you find it more readable though. Just a suggestion 😄 

```suggestion
spacing: ksSpacing,
```

Reply:
noice catch will update the pr